modules/jupyter-widget/src/utils.js

```diff
@@ -67,3 +79,10 @@ export function initDeck({mapboxApiKey, container, jsonInput, tooltip, onComplet
     });
   });
 }
+
+function wrapWarn(warnFunction, messageHandler) {
+  return (message, err) => {
```

Comment:
These argument names do not reflect what is passed to `log.warn`. Use `...args`.